app/package.json

```diff
@@ -3,7 +3,8 @@
   "license": "MPL-2.0",
   "private": true,
   "workspaces": [
-    "experimenter/static/*"
+    "experimenter/static/*",
+    "@experimenter/nimbus-ui"
```

Comment:
I think this might need to be the directory, not the package name.

Reply:
Oh yeah maybe it just needs to be `experimenter/nimbus-ui/`?

Reply:
Yeah, I tried the path at first and then flailed around and ended up with that there, my problem ended up being that it halted installing `nimbus-shared` since that requires Node 14. It works now that I've upgraded locally.